tasks/sysctl.yml

```diff
@@ -37,5 +37,6 @@
     state: present
     reload: yes
     ignoreerrors: yes
+  ignore_errors: yes
```

Comment:
Already mentioned by @rndmh3ro, but `ignore_errors: yes` at the task level can have unintended side-effects. :-1: